tests/integration/test_logcollector/test_location/test_exclude.py

```diff
@@ -0,0 +1,255 @@
+        {'location': '/tmp/wazuh-testing/test*',
+         'files': ['/tmp/wazuh-testing/test1.txt', '/tmp/wazuh-testing/test1.log', '/tmp/wazuh-testing/test2.log'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/test*', 'expected_matches': ['none'],
+         'description': 'Testing right wildcard, right exclude'},
+        {'location': '/tmp/wazuh-testing/*test.txt',
+         'files': ['/tmp/wazuh-testing/1test.txt', '/tmp/wazuh-testing/2test.txt', '/tmp/wazuh-testing/test.txt'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/test*',
+         'expected_matches': ['/tmp/wazuh-testing/1test.txt', '/tmp/wazuh-testing/2test.txt'],
+         'description': 'Testing left wildcard, right exclude'},
+        {'location': '/tmp/wazuh-testing/*test*',
+         'files': ['/tmp/wazuh-testing/1test1.txt', '/tmp/wazuh-testing/1test1.log', '/tmp/wazuh-testing/2test2.log',
+                   '/tmp/wazuh-testing/test2.txt'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/test*',
+         'expected_matches': ['/tmp/wazuh-testing/1test1.txt', '/tmp/wazuh-testing/1test1.log',
+                              '/tmp/wazuh-testing/2test2.log'],
+         'description': 'Testing right and left wildcard, right exclude'},
+        {'location': '/tmp/wazuh-testing/test*',
+         'files': ['/tmp/wazuh-testing/test1.txt', '/tmp/wazuh-testing/test1.log', '/tmp/wazuh-testing/test2.log'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/*test*',
+         'expected_matches': ['none'],
+         'description': 'Testing right wildcard, right and left exclude'},
+        {'location': '/tmp/wazuh-testing/*test.txt',
+         'files': ['/tmp/wazuh-testing/1test.txt', '/tmp/wazuh-testing/2test.txt'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/*test*',
+         'expected_matches': ['none'],
+         'description': 'Testing left wildcard, right and left exclude'},
+        {'location': '/tmp/wazuh-testing/*test*',
+         'files': ['/tmp/wazuh-testing/1test1.txt', '/tmp/wazuh-testing/1test1.log', '/tmp/wazuh-testing/2test2.log',
+                   '/tmp/wazuh-testing/test2.txt'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/*test*',
+         'expected_matches': ['none'],
+         'description': 'Testing right and left wildcard, right and left exclude'},
+        {'location': '/tmp/wazuh-testing/*',
+         'files': ['/tmp/wazuh-testing/1test1.txt', '/tmp/wazuh-testing/1test1.log', '/tmp/wazuh-testing/2test2.log',
+                   '/tmp/wazuh-testing/test2.txt'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/*',
+         'expected_matches': ['none'],
+         'description': 'Testing wildcard location'}
+
+    ]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOCATION'], x['LOG_FORMAT']}" for x in parameters]

```

Comment:
```suggestion
configuration_ids = [f"{x['LOCATION']}_{x['LOG_FORMAT']}" for x in parameters]
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/9a947325253e65b891a1766759b8bfb5134a15c7